flow/connectors/postgres/cdc.go

```diff
@@ -504,7 +516,7 @@ func (p *PostgresCDCSource) processMessage(batch *model.CDCRecordStream, xld pgl
 		if p.relationMessageMapping[msg.RelationID] == nil {
 			p.relationMessageMapping[msg.RelationID] = convertRelationMessageToProto(msg)
 		} else {
-			return p.processRelationMessage(xld.WALStart, convertRelationMessageToProto(msg))
+			return p.processRelationMessage(currentClientXlogPos, convertRelationMessageToProto(msg))
```

Comment:
Maybe comment that xld.WALStart is not available to relation message. Are there any docs to reference? Can always link to pg code